checkov/bicep/graph_builder/graph_to_tf_definitions.py

```diff
@@ -4,13 +4,12 @@
 from pathlib import Path
 from typing import Any, TYPE_CHECKING, cast
 
-from pycep.typing import BicepJson
-
 from checkov.bicep.graph_builder.graph_components.block_types import BlockType, BlockTypeAlias
 from checkov.bicep.graph_builder.local_graph import BicepElements, BicepElementsAlias
 
 if TYPE_CHECKING:
     from checkov.bicep.graph_builder.graph_components.blocks import BicepBlock
+    from pycep.typing import BicepJson
```

Comment:
why do we need it inside the if statement?
I thought that the "if TYPE_CHECKING" is for avoiding circular dependences

Reply:
basically during runtime the `if` block is ignored, so none of the imports is actually imported, which of course benefits performance and in some cases helps to prevent circular dependencies. Sadly the plugin can't find circular dependencies, it just helps to optimize the imports, which are need to type hints.

Reply:
thanks!